config/devel.yaml

```diff
@@ -3,15 +3,15 @@ PLAINTEXT_VARIABLES:
     AWS_REGION: us-east-1
     SIERRA_DB_PORT: 1032
     SIERRA_DB_NAME: iii
-    REDSHIFT_DB_NAME: qa
-    REDSHIFT_TABLE: patron_info_qa
+    REDSHIFT_DB_NAME: dev
+    REDSHIFT_TABLE: patron_info_dev
     GEOCODER_API_BASE_URL: https://geocoding.geo.census.gov/geocoder/geographies/addressbatch
     GEOCODER_API_BENCHMARK: Public_AR_Current
     GEOCODER_API_VINTAGE: Current_Current
     PATRON_INFO_SCHEMA_URL: https://qa-platform.nypl.org/api/v0.1/current-schemas/PatronInfo
-    KINESIS_BATCH_SIZE: 20
-    S3_BUCKET: bic-poller-states-qa
-    S3_RESOURCE: patron_info_poller_state.json
+    KINESIS_BATCH_SIZE:
+    S3_BUCKET:
+    S3_RESOURCE:
```

Comment:
Just to clarify, was this removed bc there are no dev Kinesis streams? (i.e. we don't want anything to be sent out from local runs)

Reply:
Yeah there is no dev Kinesis/S3 and I wanted to make sure people didn't accidentally overwrite the QA S3 buckets/write to the QA Kinesis stream. Before these had to be filled because even if IGNORE_KINESIS was true the Kinesis client would still get created even though it wasn't used.